Phonebook.Frontend/src/app/pages/dashboard/components/overview/dashboard.component.ts

```diff
@@ -59,6 +68,19 @@ export class DashboardComponent implements OnInit, OnDestroy {
           this.drawerOpen = this.store.selectSnapshot(AppState.recentPeopleDrawer);
         }
       });
+    this.currentUserService
+    .getCurrentUser()
+    .pipe(untilComponentDestroyed(this))
+    .subscribe(
+      user => {
+        if (user != null && this.bookmarkedPersons.length === 0) {
+          this.router.navigate(['/dashboard/team']);
+        }
+      },
+      error => {
+        this.currentUser = null;
+      }
+    );
   }
 
   public changeOrder() {
```

Comment:
Not needed anymore?

Reply:
Please remove every function or variable that was previously needed for the bookmarks but is now moved to the bookmarks.component

Reply:
Actually it's still needed. Since it is supposed to redirect to the Team when there are no Bookmarks, it needs to know how many bookmarks there are. If we put that bit of code into a routerGuard as you suggested, this might not be the case anymore.

Reply:
than remove it after implementing the routerGuard